src/bin_editor/Oni2_editor.re

```diff
@@ -56,13 +56,26 @@ let init = app => {
 
   let update = UI.start(w, <Root state=currentState^ />);
 
+  let isDirty = ref(false);
   let onStateChanged = state => {
     currentState := state;
+    isDirty := true;
     GlobalContext.set({...GlobalContext.current(), state});
-
     update(<Root state />);
```

Comment:
Was this line supposed to poof?

Reply:
Indeed it was! Nice catch @izik1, thank you 👍